code/datums/helper_datums/getrev.dm

```diff
@@ -58,6 +58,9 @@
 	if(GLOB.round_id)
 		msg += "<b>Round ID:</b> [GLOB.round_id]"
 
+	msg += "<b>BYOND Version:</b> [world.byond_version].[world.byond_build]"
+	if(DM_VERSION != world.byond_version || DM_BUILD != world.byond_build)
+		msg += "<b>DM Version:</b> [DM_VERSION].[DM_BUILD]"

```

Comment:
Change this to `Compiled with BYOND Version: ` otherwise we'll forever be answering questions about this.